### PR TITLE
[three] r158

### PR DESCRIPTION
**Pull request by Methuselah96**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/three-types/three-ts-types/issues/631
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by jakebailey** [comment]
Looking into that failure.

File: types/three/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/three",
-    "version": "0.157.9999",
+    "version": "0.158.0",
```
Comment (jakebailey) [inline types/three/package.json:4]:
```suggestion
    "version": "0.158.9999",
```

The `.9999` is required; it's to ensure that this package is "newest" in the package graph within the repo for this version (so npm isn't used). This error is unfortunately fatal but we don't error out first thing; will see about fixing that now that loading is lazier.

Comment (Methuselah96) [reply types/three/package.json]: Aha, got it, thanks!

Comment (jakebailey) [reply types/three/package.json]: Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/789 which would have exposed this sooner.

**Comment by DangerBotOSS** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>dprint failed to execute</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.ts`.
  </td>
    </tr>
  </tbody>
</table>



Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## three ([<kbd>unpkg</kbd>](https://unpkg.com/browse/three@latest/))
was missing the following properties:
1. CompressedCubeTexture

## Error TypeError
```
Cannot use 'in' operator to search for '0' in Compiling https://plugins.dprint.dev/json-0.19.0.wasm
Compiling https://plugins.dprint.dev/typescript-0.88.3.wasm
No files found to format with the specified plugins at /home/runner/work/DefinitelyTyped/DefinitelyTyped. You may want to try using `dprint output-file-paths` to see which files it's finding.

No files found to format with the specified plugins at /home/runner/work/DefinitelyTyped/DefinitelyTyped. You may want to try using `dprint output-file-paths` to see which files it's finding.
TypeError: Cannot use 'in' operator to search for '0' in Compiling https://plugins.dprint.dev/json-0.19.0.wasm
Compiling https://plugins.dprint.dev/typescript-0.88.3.wasm
No files found to format with the specified plugins at /home/runner/work/DefinitelyTyped/DefinitelyTyped. You may want to try using `dprint output-file-paths` to see which files it's finding.

No files found to format with the specified plugins at /home/runner/work/DefinitelyTyped/DefinitelyTyped. You may want to try using `dprint output-file-paths` to see which files it's finding.
    at __spreadArray (dangerfile.ts:4:23)
    at Object.<anonymous> (dangerfile.ts:106:33)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at requireFromString (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/require-from-string@2.0.2/node_modules/require-from-string/index.js:28:4)
    at /home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/danger@11.3.0_@octokit+core@3.6.0/node_modules/danger/distribution/runner/runners/inline.js:161:68
    at step (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/danger@11.3.0_@octokit+core@3.6.0/node_modules/danger/distribution/runner/runners/inline.js:52:23)
    at Object.next (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/danger@11.3.0_@octokit+core@3.6.0/node_modules/danger/distribution/runner/runners/inline.js:33:53)
    at /home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/danger@11.3.0_@octokit+core@3.6.0/node_modules/danger/distribution/runner/runners/inline.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/.pnpm/danger@11.3.0_@octokit+core@3.6.0/node_modules/danger/distribution/runner/runners/inline.js:23:12)
```
### Dangerfile
```
1| import fs = require("fs");
2| import os = require("os");
3| import path = require("path");
------------------------^
4| import cp = require("child_process");
5| // Removed import { danger, fail, markdown } from "danger";
6| const suggestionsDir = [os.homedir(), ".dts", "suggestions"].join("/");
7| const lines: string[] = [];
8| const missingProperty = /module exports a property named '(.+?)', which is missing/;
9| 
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 502a7c91cb6dbae486caf9489465c763dc2e72ae
</p>

**Comment by jakebailey** [comment]
Okay, fixed...

**Review by jakebailey — APPROVED** [review]

**Comment by Methuselah96** [comment]
Ready to merge

**Pull request merged**